addons/cert-manager/template/testgrid/k8s-docker.yaml

```diff
@@ -0,0 +1,11 @@
+kubernetes:
+  version: "latest"
+weave:
+  version: "latest"
+rook:
+  version: "latest"
```

Comment:
rook isn't required here

Reply:
I left Rook (or some other storage class) in all of them because when the TestGrid runner executes the part of the script that needs to create a troubleshooting bundle, it was dying without storage for some reason. Part of the reason for moving to these tempalte. Is there a better way around this?

Reply:
this one: https://github.com/replicatedhq/kURL/blob/db70ebca6bf8eca87aae75a515189de69947d507/testgrid/tgrun/pkg/instances/k8s119_minimal_containerd137.go#L9

Reply:
You called it: https://testgrid.kurl.sh/run/certManger-wo-rook

Reply:
removing rook from the mentioned addons.